aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -13,7 +13,56 @@ uid: fundamentals/servers/kestrel
 
 By [Tom Dykstra](https://github.com/tdykstra), [Chris Ross](https://github.com/Tratcher), and [Stephen Halter](https://twitter.com/halter73)
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the web server that's included and enabled by default in ASP.NET Core project templates.
+
+Kestrel supports the following scenarios:
+
+* HTTPS
```

Comment:
May eventually want a doc on specifically HTTPS?

Reply:
Maybe. Currently it's just part of endpoints.md, but that's already very large. I think we're going to eventually split that as well.